src/Git/WorkingCopy.php

```diff
@@ -60,21 +60,26 @@ public function addFork($fork_url)
         $this->fork = new Remote($fork_url);
         $this->fork->addAuthentication($this->api);
 
+        $this->addRemote($this->fork->url(), 'fork');
+
         return $this;
     }
 
     /**
      * createFork creates a new secondary repository copied from
      * the current repository, and sets it up as a fork per 'addFork'.
      */
-    public function createFork($forked_project_name, $forked_org = '', $branch = '')
-    {
-        $result = $this->api->gitHubAPI()->api('repo')->create(
-            $forked_project_name,
-            '',
-            '',
-            true,
-            $forked_org
+    public function createFork($forked_project_name, $forked_org = null, $branch = '')
+    {
+        [$org, $project_name] = explode('/', $this->remote->projectWithOrg());
+        $result = $this->api->gitHubAPI()->api('repo')->forks()->create(
```

Comment:
Creating a PR was failing because the new repo wasn't actually a fork and you should use a fork to create a PR from a remote repo (maybe something changed recently in Github API?)

Reply:
Could be -- I think it used to let me do that. I don't think this feature is being used by any of the active updatinator jobs, though, so it could have broken some time ago.